src/storage/sstable.rs

```diff
@@ -40,7 +40,9 @@ impl SSTable {
             .open(&path)?;
         let mut offset = 0usize;
         for (kv_i, (k, v)) in memtable.iter().enumerate() {
-            let delta_offset = serde::serialize_kv(k, v, &mut file)?;
+            let mut delta_offset = 0usize;
```

Comment:
it might simplify some call-sites to expose a function in `serde` like this:

``` rust
serde::serialize_kv(k, v, &mut file) -> Result<int> {
  i = k.ser(file)?;
  i += v.ser(file)?;
  i
}
```